### PR TITLE
add jsonb's index use case

### PR DESCRIPTION
**Pull request by st1page**
## Info

- **Description**

add a example for index on jsonb column with in section "Indexes on expressions"

- **Notes**

  - [ Include any supplementary context or references here. ]

- **Related code PR**

  - [ Provide a link to the relevant code PR here, if applicable. ]

- **Related doc issue**
  
  Resolves [ Provide a link to the relevant doc issue here, if applicable. ]

## For reviewers

- **Preview**

  - [ Paste the preview link to the updated page(s) here. Edit this item after the preview site is ready. To find the updated pages, scroll down to locate and open the Amplify preview link and select the **dev** version of the documentation. ]

- **Key points**

  - [ Parts that may need revision or extra consideration. ]

## Before merging

- [ ] I have checked the doc site preview, and the updated parts look good.

- [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`CharlieSYH`, `emile-00`, & `hengm3467`).

### CODE REVIEW

**Review by chenzl25 — APPROVED** [review]
LGTM

File: docs/sql/commands/sql-create-index.md [hunk 1]
```diff
@@ -181,6 +181,21 @@ then you might want to create an index like the following to improve the perform
 CREATE INDEX people_names ON people ((first_name || ' ' || last_name));
 ```
 
+This syntax is quite useful when using a semi-structured table with [JSONB](/sql/data-types/data-type-jsonb.md) datatype. Here is an example of create an index on a jsonb's field.
```
Comment (WanYixian) [inline docs/sql/commands/sql-create-index.md]:
```suggestion
This syntax is quite useful when using a semi-structured table with [JSONB](/sql/data-types/data-type-jsonb.md) datatype. Here is an example of creating an index on a field within the JSONB column.
```

**Review by WanYixian — APPROVED** [review]

File: docs/sql/commands/sql-create-index.md (hunk 1 above)
Comment (WanYixian) [reply docs/sql/commands/sql-create-index.md]: Others LGTM!

**Comment by st1page** [comment]
So sorry for forget this PR for a long time 🥵 and the PR will be replaced with https://github.com/risingwavelabs/risingwave-docs/pull/1977

**Pull request closed without merging**